extensions/laravel-livewire/package.json

```
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "laravel-livewire",
+  "title": "Laravel Livewire",
+  "description": "Quickly search through Laravel Livewire documentation and open it in the browser",
+  "icon": "livewire-icon-transparent.png",
+  "author": "tafhyseni",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Documentation",
+      "subtitle": "Laravel Livewire",
+      "description": "Search Livewire",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "livewireVersion",
+          "type": "dropdown",
+          "required": true,
+          "title": "Livewire Version",
+          "description": "Specify the Laravel livewire version it will search the docs for.",
+          "default": "1.x",
```

```suggestion
          "default": "2.x",
```
Wouldn't it make more sense to have the newest as default?